gen/cow/say.hoon

```diff
@@ -0,0 +1,60 @@
+::  The classic unix "cowsay" implemented in hoon
+::
+::  To see a cow say a quote from gen/cow/quotes.txt run:
+::
+::  +cow/say
+::
+/-  sole
+/+   *generators, pretty-file
+:-  %ask
+|=  [[now=@da eny=@uvJ bec=beak] ~ ~]
+=/  max-length  80
+=/  file-path  `path`/(scot %p p.bec)/[q.bec]/(scot r.bec)/gen/cow/quotes/txt
```

Comment:
`=/` actually allows you to do the cast like so: `=/ file-path=path /what/ever`.

It's generally good practice to add a type to every use of `=/`, but since you use so many of them _and they're nearly all `tape`_, I'd say it's fine as-is. Maybe add a type for the exceptions, but that's up to you.